data/src/main/java/io/gnosis/data/models/transaction/TransactionDetails.kt

```diff
@@ -47,27 +50,27 @@ sealed class DetailedExecutionInfo(
     @JsonClass(generateAdapter = true)
     data class MultisigExecutionDetails(
         @Json(name = "submittedAt")
-        val submittedAt: Date,
+        val submittedAt: Date = Date(),
         @Json(name = "nonce")
         val nonce: BigInteger,
         @Json(name = "safeTxHash")
-        val safeTxHash: String,
+        val safeTxHash: String = "",
         @Json(name = "signers")
-        val signers: List<Solidity.Address>,
+        val signers: List<Solidity.Address> = emptyList(),
         @Json(name = "confirmationsRequired")
-        val confirmationsRequired: Int,
+        val confirmationsRequired: Int = 0,
         @Json(name = "confirmations")
-        val confirmations: List<Confirmations>,
+        val confirmations: List<Confirmations> = emptyList(),
         @Json(name = "executor")
-        val executor: Solidity.Address?,
+        val executor: Solidity.Address? = null,
         @Json(name = "safeTxGas")
-        val safeTxGas: BigInteger,
+        val safeTxGas: BigInteger = BigInteger.ZERO,
         @Json(name = "baseGas")
-        val baseGas: BigInteger,
+        val baseGas: BigInteger = BigInteger.ZERO,
         @Json(name = "gasPrice")
-        val gasPrice: BigInteger,
+        val gasPrice: BigInteger = BigInteger.ZERO,
         @Json(name = "gasToken")
-        val gasToken: Solidity.Address
+        val gasToken: Solidity.Address = "0".asEthereumAddress()!!
```

Comment:
Why do you need default values for all he fields in `MultisigExecutionDetails` and `TransactionDetails`?

Reply:
I used default values, so I could make the invocation shorter.